modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsControllerEpicsCommon.scala

```diff
@@ -276,14 +276,11 @@ object TcsControllerEpicsCommon {
     : List[BaseEpicsTcsConfig => F[BaseEpicsTcsConfig]] = List(
       setMountGuide(Lens.id)(subsystems, current.telescopeGuideConfig.mountGuide, demand.gc.mountGuide),
       setM1Guide(Lens.id)(subsystems, current.telescopeGuideConfig.m1Guide, demand.gc.m1Guide),
-      setM2Guide(Lens.id)(subsystems, current.telescopeGuideConfig.m2Guide, demand.gc.m2Guide),
-      setPwfs1(Lens.id)(subsystems, current.pwfs1.detector, demand.gds.pwfs1.detector),
-      setPwfs2(Lens.id)(subsystems, current.pwfs2.detector, demand.gds.pwfs2.detector),
-      setOiwfs(Lens.id)(subsystems, current.oiwfs.detector, demand.gds.oiwfs.detector)
```

Comment:
This is another bug I found thanks to the new unit tests. Guiders should keep reading out during offsets.